packages/client/src/client.ts

```diff
@@ -16,13 +17,25 @@ export class Client {
   static *create(url: string): Operation<Client> {
     let socket = new w3cwebsocket(url) as WebSocket;
 
+    yield spawn(function* detectStartupError(): Operation<void> {
+      let [error] = yield once(socket, 'error');
+      
+      if (isYaetiError(error)) {
```

Comment:
I'd call this `isErrorEvent` since Yaeti is just the `nodejs` error event emulator for the [`ErrorEvent`](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent). When the client is running on the browser, it will actually be an `ErrorEvent`

Reply:
Worth noting that on the web, this will _always_ be an error event.